server/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
```

Comment:
Utils means nothing, is the same thing of calling it "stuff", or "magic". In this case, we can create a "markdown" package, if you want, and put here any utilities functions related to markdown. But creating a `utils` package is a bad idea in general.